daprdocs/content/en/developing-applications/building-blocks/service-invocation/howto-invoke-services-grpc.md

```diff
@@ -0,0 +1,238 @@
+---
+type: docs
+title: "How-To: Invoke services using gRPC"
+linkTitle: "How-To: Invoke services using gRPC"
+description: "Call between services using service invocation"
+weight: 3000
+---
+
+This article describe how to deploy services each with an unique application ID, so that other services can discover and call endpoints on them using service invocation API.
```

Comment:
I feel that this overview should explain more on what you are covering here. Add a sentence on the scenario, for example this is used when you want to call an existing gRPC application that does not use Dapr. People always want clarification on what the scenario this applies to.

Reply:
Done